site/class-1-bar/readme.md

```diff
@@ -1,9 +0,0 @@
-# Bar Chart (Class 1) and Readme (Class 2)
```

Comment:
We talked about this on slack, you’re overwriting our file here. Maybe it’s simpler to start over again?